src/Illuminate/Mail/Transport/SesTransport.php

```diff
@@ -46,25 +52,32 @@ public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = nul
         $this->beforeSendPerformed($message);
 
         try {
-            $result = $this->ses->sendRawEmail(
-                array_merge(
-                    $this->options, [
-                        'Source' => key($message->getSender() ?: $message->getFrom()),
-                        'RawMessage' => [
-                            'Data' => $message->toString(),
-                        ],
-                    ]
-                )
-            );
+            /**
+             * Batch up API calls to adhere to the recipient limit of the service.
+             * See Also: https://docs.aws.amazon.com/aws-sdk-php/v2/api/class-Aws.Ses.SesClient.html#_sendRawEmail.
+             */
+            foreach (collect(array_merge($message->getTo(), $message->getCc(), $message->getBcc()))->chunk(self::RECIPIENT_LIMIT) as $recipients) {
+                $result = $this->ses->sendRawEmail(
+                    array_merge(
+                        $this->options, [
+                            'Source' => key($message->getSender() ?: $message->getFrom()),
+                            'RawMessage' => [
+                                'Data' => $message->toString(),
+                            ],
+                            'Destinations' => $recipients,
+                        ]
+                    )
+                );
+
+                $messageId = $result->get('MessageId');
+
+                $message->getHeaders()->addTextHeader('X-Message-ID', $messageId);
```

Comment:
This will only add the message ID of the last message, right, throwing out the earlier IDs?

Reply:
If it has to be a singular value and most RFC docs specify it will be when present, what do we want to capture here? 

I could aggregate them up and add a CSV of the values? I'm not sure who may inspect/capture this or what processing they may or may not do today. I could also just capture the first one and drop any subsequent batches (to keep it behaving exactly like today, but sending > 50 correctly and losing those extra message IDs).

A single value (first, last, any) is the same "API" as today. First represents identical. Last would be a slight change. A CSV is a change in the API. In 9.x, this code is gone altogether, but I didn't want to change the "API" .